Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -10,6 +10,24 @@ public static class ArrayOperations
         public static void Sort(int[] array)
         {
             // ToDo: implement.
+            if(array != null || array.Length >= 1)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either reverse the equality operator in the 'array' null test, or reverse the logical operator that follows it.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4940124)